example-hello/src/main/resources/application.properties

```diff
@@ -1,2 +1,2 @@
-server.port=1111
+server.port=1740
```

Comment:
This was a randomly generated number that I checked against [Wikipedia](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers) and confirmed it wasn't in widespread use.